quickstart/README.md

```diff
@@ -7,17 +7,32 @@
 
 ## Lab
 
+The container config of [Lobsters](https://github.com/jcs/lobsters) is
+[here](lobsters/). It is a test configuration with a local sqlite db
+inside the single container.
+
 Run the Lobsters app container
 
-<!-- dont-START bash -->
+<!-- START bash -->
 ```
 kubectl run lobsters --image=gcr.io/google-samples/lobsters:latest
 ```
-<!-- dont-END bash -->
+<!-- END bash -->
 
-<!-- Test this in the meantime START bash
-kubectl run nginx --image=nginx
-  END bash -->
+```
+deployment "lobsters" created
+```
+
+Get deployments
+
+```
+kubectl get deployments
```

Comment:
Right, this is what we settled on for samples in the k8s repo, two seperate text boxes for the command and output, to avoid the $ c&p issue.

I think if we are consistent with always showing the command and output, it will be clear. What do you think? an explicit "run:" and "output:" ahead of each block?